src/servletTutorial/manage/BookManagement.java

```diff
@@ -50,4 +37,40 @@ public static void deleteBook(int bookId) {
 			}
 		}
 	}
+
+	public static Book getBookById(int bookId) {
+
+		for (Book book : listBooks) {
+			if (book.getBookId() == bookId) {
+				return book;
+			}
+		}
+		return null;
```

Comment:
The `get` method will be throw exception if it doesnot exist. The `fetch` method will return null if it doesnot exist.